packages/passport/sdk/src/zkEvm/zkEvmProvider.ts

```diff
@@ -256,28 +243,25 @@ export class ZkEvmProvider implements Provider {
         }
       }
       case 'eth_accounts': {
-        const zkEvmAddress = await this.#getZkEvmAddress();
+        const zkEvmAddress = zkEvmUser?.zkEvm.ethAddress;
```

Comment:
If `zkEvmUser` is null, then `zkevm` will also be null, and so this will throw a `cannot read properties of undefined` error

Reply:
just missing an optional chain, pushed https://github.com/immutable/ts-immutable-sdk/pull/1965/commits/b9397aa3454878abf3dcbd718e98437982a80ff3